cookbooks/aws-parallelcluster-environment/recipes/config/export_home.rb

```diff
@@ -14,6 +14,18 @@
 
 return if on_docker?
 
+return unless node['cluster']['internal_shared_storage_type'] == 'ebs'
+
+# Check if home is a shared filesystem and return if it is because
+# there is nothing to export
+is_home_shared = false
+efs_shared_dir_array = node['cluster']['efs_shared_dirs'].split(',')
+fsx_shared_dir_array = node['cluster']['fsx_shared_dirs'].split(',')
+(efs_shared_dir_array + fsx_shared_dir_array).each do |dir|
+  is_home_shared = true if dir == "/home"
+end
+return if is_home_shared
```

Comment:
What if the /home is the mountdir of an EBS shared storage and the internal shared storage is EBS?
We would end up skipping to export the /home in that case, that is not desirable.